frontend/src/Components/ProductTypes/Products.tsx

```diff
@@ -186,6 +192,39 @@ const Products = () => {
           transformData={transformIncomePaystubsData}
           />
       )}
+
+      {(products.includes("cra_base_report") || products.includes("cra_income_insights")) && (
+        <Endpoint
+          endpoint="/cra/check_report"
+          name="CRA Base Report"
+          categories={checkReportGetCategories}
+          schema="/cra/check_report/base_report/get"
+          description="Retrieve a Consumer Report powered by Plaid Check"
+          transformData={transformCheckReportGetData}
+        />
+      )}
+
+      {(products.includes("cra_base_report") || products.includes("cra_income_insights")) && (
+        <Endpoint
+          endpoint="/cra/income_insights"
+          name="CRA Income Insights"
+          categories={checkReportInsightsCategories}
+          schema="/cra/check_report/income_insights/get"
+          description="Retrieve cash flow information from your user's banks"
+          transformData={transformIncomeInsightsData}
+        />
+      )}
+
+      {products.includes("cra_partner_insights") && (
+        <Endpoint
+          endpoint="/cra/partner_insights"
```

Comment:
`/cra/check_report/partner_insights`

Reply:
updating to `/cra/get_partner_insights/`